### PR TITLE
Copter: _batt_voltage_filt bug fix and other small changes

### PR DESCRIPTION
**Pull request by jschall**

### CODE REVIEW

**Comment by rmackay9** [comment]
merged, thanks!

**Comment by magicrub** [comment]
LowPassFilter.h line 116 needs a divide by zero check

File: libraries/Filter/LowPassFilter.h [hunk 1]
```diff
@@ -105,10 +108,13 @@ void LowPassFilter<T>::set_time_constant(float time_step, float time_constant)
 {
     // avoid divide by zero
     if (time_constant + time_step <= 0.0f) {
```
Comment (magicrub) [inline libraries/Filter/LowPassFilter.h]: needs:
|| time_constant == 0

**Comment by rmackay9** [comment]
ok, thanks for that.  How about this fix?
https://github.com/diydrones/ardupilot/commit/ca92821445b4de29991d1f25388d802aa90b5474

**Pull request closed without merging**